### DISCUSSION TITLE
[2230650]TEVI 本体

### DISCUSSION THREAD

**Discussion by lueluo** [pjy612/SteamManifestCache#172] [Share]
`appid;2230650`
keys;
```
2230651;cf4bd9819dcbe55a664cd8774f36ed68fb0bcff0071c1600b939704a3b7df38f
```
email:jueluo12@163.com
code:AC90A6D3EDDB685C

[tevi.zip](https://github.com/pjy612/SteamManifestCache/files/13768830/tevi.zip)

**Comment by pjy612** [comment]
查收
